x/ccv/provider/types/msg.go

```diff
@@ -325,24 +336,28 @@ func (msg MsgUpdateConsumer) Route() string { return RouterKey }
 // ValidateBasic implements the sdk.Msg interface.
 func (msg MsgUpdateConsumer) ValidateBasic() error {
 	if err := ValidateConsumerId(msg.ConsumerId); err != nil {
-		return err
+		return errorsmod.Wrapf(ErrInvalidMsgUpdateConsumer, "ConsumerId: %s", err.Error())
+	}
+
+	if err := ccvtypes.ValidateAccAddress(msg.NewOwnerAddress); err != nil {
```

Comment:
If my understanding is correct, `NewOwnerAddress` can be empty, in which case we do not overwrite the owner address of a consumer chain. What happens with this validation if `msg.NewOwnerAddress` is empty? Note that `NewOwnerAddress` is already validate inside `UpdateConsumer` as such:
```
if strings.TrimSpace(msg.NewOwnerAddress) != "" {
    if _, err := k.accountKeeper.AddressCodec().StringToBytes(msg.NewOwnerAddress); err != nil {
	return &resp, errorsmod.Wrapf(types.ErrInvalidNewOwnerAddress, "invalid new owner address %s", msg.NewOwnerAddress)
    }
}
```

Reply:
This is a very good point. I'll remove the validation from here.